modify_posts/create-post.js

```diff
@@ -0,0 +1,64 @@
+const apiUrl = "https://api.noroff.dev";
+const postsUrl = "/api/v1/social/posts";
+const allPostsUrl = apiUrl + postsUrl;
+
+const upload = document.querySelector(".upload");
+upload.addEventListener("click", onClick);
+
+const imageContainer = document.querySelector(".image");
+const titleContainer = document.querySelector(".title");
+const bodyContainer = document.querySelector(".body");
+
+/**
+ * This will create a new post
+ * in the form with image, title
+ * and body inputs that will
+ * be posted to the API
+ * @param {string} url The post url
+ * @param {string} image The post image
+ * @param {string} title The post title
+ * @param {string} body The post body
+ */
+async function createPost(url, image, title, body) {
+  try {
+    const token = localStorage.getItem("accessToken");
+    const getData = {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${token}`,
+      },
+      body: JSON.stringify({
+        media: image,
+        title: title,
+        body: body,
+      }),
+    };
+    const response = await fetch(url, getData);
+  } catch (error) {}
```

Comment:
This looks incomplete. You have to do something in the body of the catch block. You can return the error. And if you don't need it, you can remove it.